cis/schema.json

```diff
@@ -206,22 +216,22 @@
         "picture": {
             "type": "string",
             "format": "uri",
-            "description": "(Informational) The URL of an image representing this user."
+            "description": "The URL to an image representing this user."
         },
         "shirtSize": {
             "type": "string",
-            "description": "(Informational) Shirt size for this user. Format undefined."
+            "description": "Shirt size for this user. Format undefined."
         },
         "groups": {
             "type": "array",
-            "description": "(Informational) Deprecated. Use `authoritativeGroups` instead.",
+            "description": "Informational group membership for the user. Non-prefixed groups are what is known as Mozilla LDAP groups. Groups prefixed by a provider name and underscore are provided by a specific group engine.\nFor example `providername_groupone` is provided by `providername`.\nThese groups are provided `as-is` and you should ensure it contains the members you expect.",
```

Comment:
`groups` are used for access control. how much of a guarantee groups are properly *managed* is up to you, authoritative groups have a better guarantee. Both are needed in practice.